src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/Models/Cohort/Creation/CohortCreationTestFixture.cs

```diff
@@ -136,17 +138,33 @@ public void VerifyCohortBelongsToProvider()
             Assert.AreEqual(Provider.UkPrn, Cohort.ProviderId);
         }
 
+        public void VerifyLastUpdatedFieldsAreSetForParty(Party modifyingParty)
```

Comment:
It doesn't look like this method is called.

Reply:
It's called by WhenCohortIsCreated.ThenCohortHasLastedUpdatedBy